data/json/items/melee/knives_kitchen.json

```diff
@@ -164,9 +164,9 @@
     "longest_side": "30 cm",
     "price": 3500,
     "price_postapoc": 50,
-    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "uneven" },
-    "bashing": 6,
-    "cutting": 20,
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "neutral" },
+    "bashing": 4,
+    "cutting": 15,
```

Comment:
Any reason to make this less effective than a chef knife? A cleaver is pretty hefty.

Reply:
It was 20 before, @anothersimulacrum requested to lower the damage so I brought it in-line with the vegetable cleaver.

Reply:
If that's consistent with the veg it is probalby ok.